test/new-e2e/pkg/environments/aws/docker/host.go

```diff
@@ -50,6 +51,15 @@ func newProvisionerParams() *ProvisionerParams {
 	}
 }
 
+func GetProvisionerParams(opts ...ProvisionerOption) *ProvisionerParams {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
// GetProvisionerParams creates a ProvisionerParams instance and applies ProvisionerOption to it
func GetProvisionerParams(opts ...ProvisionerOption) *ProvisionerParams {
```